src/FlaskRTBCTF/users/routes.py

```diff
@@ -48,6 +49,18 @@ def register():
 
         db.session.add(log)
         db.session.add(user)
+        db.session.flush()
+
+        Machines = Machine.query.all()
+        for machine in Machines:
```

Comment:
we don't need to initialize rows for every User & Machine. It would be created only when user submits flag, right or wrong.